src/main/java/net/minecraftforge/server/permission/PermissionAPI.java

```diff
@@ -108,7 +109,13 @@ public static ResourceLocation getActivePermissionHandler()
      */
     public static void initializePermissionAPI()
     {
-        if (PermissionAPI.activeHandler != null) throw new IllegalStateException("Tried to initialize PermissionAPI multiple times!");
+        if(StackWalker.getInstance(StackWalker.Option.RETAIN_CLASS_REFERENCE).getCallerClass() != ServerLifecycleHooks.class)
```

Comment:
Formatting: space after `if` and before opening parenthesis.
```suggestion
        if (StackWalker.getInstance(StackWalker.Option.RETAIN_CLASS_REFERENCE).getCallerClass() != ServerLifecycleHooks.class)
```
Additionally, it may be good to store the calling class and, if not `ServerLifecycleHooks`, log the name of the class.